make/Images.gmk

```diff
@@ -30,6 +30,7 @@ include MakeBase.gmk
 include Execute.gmk
 include Modules.gmk
 include Utils.gmk
+include ToolsJdk.gmk
```

Comment:
This is probably no longer needed. I'll remove in a later update.

Reply:
Yes, please do.